runelite-client/src/main/java/net/runelite/client/plugins/prayer/PrayerConfig.java

```diff
@@ -32,18 +32,18 @@
 public interface PrayerConfig extends Config
 {
 	@ConfigItem(
-		position = 0,
-		keyName = "prayerflick",
-		name = "Prayer flick helper",
-		description = "Enable the prayer flick helper"
+		position = 1,
+		keyName = "prayerFlickLocation",
+		name = "Pray flick location",
+		description = "Choose where to display the prayer flick helper"
 	)
-	default boolean prayerFlickHelper()
+	default PrayerFlickLocation prayerFlickLocation()
 	{
-		return true;
+		return PrayerFlickLocation.NONE;
 	}
 
 	@ConfigItem(
-		position = 1,
+		position = 2,
```

Comment:
Remove this and below re-numbering

Reply:
As in revert? or just remove all position numbering from 46 down and leave line 35 as is?